test_runner/fixtures/neon_tenant.py

```diff
@@ -0,0 +1,352 @@
+from __future__ import annotations
```

Comment:
this is weird, are annotations still in future? Aah, nope it means the py3.7 changes to type hints: https://stackoverflow.com/questions/61544854/from-future-import-annotations/61544901#61544901